src/_i18n/es.yml

```diff
@@ -141,9 +141,14 @@ publications-post:
 services:
   title: Software de calidad entregado efectivamente
   subtitle: Lideres reconocidos en continuous delivery y prácticas de calidad y pericia.
+  we-acd: We advise, we coach, we deliver
```

Comment:
Asesoramos, ensenamos, desarrollamos.   ????

Reply:
I'm leaving it for now until we find a better alternative, also I'm not sure if it's our new motto and we should translate it.